projects/components/src/table/cells/table-cell-injection.ts

```diff
@@ -1,12 +1,16 @@
 import { InjectionToken, Injector } from '@angular/core';
 import { TableColumnConfig, TableRow } from '../table-api';
 import { TableCellParserBase } from './table-cell-parser-base';
+import { CoreTableCellCsvGeneratorType } from './types/core-table-cell-csv-generator-type';
 
 export const TABLE_COLUMN_CONFIG: InjectionToken<TableColumnConfig> = new InjectionToken('TABLE_COLUMN_CONFIG');
 export const TABLE_COLUMN_INDEX: InjectionToken<TableColumnConfig> = new InjectionToken('TABLE_COLUMN_INDEX');
 export const TABLE_DATA_PARSER: InjectionToken<TableCellParserBase<unknown, unknown, unknown>> = new InjectionToken(
   'TABLE_DATA_PARSER'
 );
+export const TABLE_CELL_CSV_GENERATOR: InjectionToken<
```

Comment:
are the changes in table cell renderers still needed?

Reply:
not required. removed.